guide/popular-topics/errors.md

```diff
@@ -1,42 +1,42 @@
 # Errors
 
-There is no doubt that you have encountered errors while making bots. While errors are very useful at warning you of what is going wrong, many people are stumped by them and how to track them down and fix them, but don't worry, we have you covered. This section will be all about how to diagnose errors, how to identify where they are coming from, and how to fix them.
+There is no doubt that you have encountered errors while making bots. While errors are instrumental at warning you of what is going wrong, many people are stumped by them and how to track them down and fix them, but don't worry, we have you covered. This section will be all about diagnosing errors, identifying where they are coming from, and fixing them.
 
 ## Types of Errors
 
 ### API Errors
 
-API Errors or DiscordAPIErrors are errors which are thrown by the Discord API when an invalid request is carried out. API Errors can be mostly diagnosed using the message that is given. They can also be further examined by seeing the http method and path used, we will explore tracking these errors down in the next section.
+API Errors or DiscordAPIErrors are thrown by the Discord API when an invalid request carries out. API Errors can be mostly diagnosed using the message that is given. You can further examine errors by inspecting the HTTP method and path used. We will explore tracking these errors down in the next section.
 
 Example: `DiscordAPIError: Cannot send an empty message`
 
-### Discord.js Errors
+### discord.js Errors
```

Comment:
```suggestion
### discord.js errors
```

In this section and the next, "errors" should be lowercase. `Error` may be a JS class but that's not exactly how it's being referred as here. The sentences "x errors are errors that are thrown when y" should be "x errors are thrown when y".